dev-env/bin/daml-docker-head

```diff
@@ -41,4 +41,4 @@ ENV PATH="/home/daml/.daml/bin:${PATH}"
 WORKDIR /home/daml
 EOF
 
-docker build ${tmpdir}
+docker build -t "digitalasset/daml-sdk:${version}" ${tmpdir}
```

Comment:
Usually the first segment (before `/`) in a Docker tag is the repo, so local tags are (by convention) not nested. Perhaps `daml-sdk-head:${version}` instead?

Also, if we're considering names, it may be worth not tagging them with 0, but instead something like `$(git show -s --format=%cd-%h --date=format:%Y%m%d-%H%M%S)`? (Only talking about changing the image name here; the build inside Docker should definitely use 0.0.0 otherwise it will take ages.)

Reply:
Not sure it makes sense to keep the docker images more persistent than SDK head itself. I think just overwriting is fine and arguably convenient since you get a fixed hash every time.